pkg/projectfile/projectfile.go

```diff
@@ -486,10 +487,11 @@ func Parse(configFilepath string) (*Project, error) {
 		return nil, errs.Wrap(err, "project.Init failed")
 	}
 
-	cfg, err := config.Get()
+	cfg, err := config.New()
```

Comment:
This may be the same problem as in the `auth.go` file.  This should create config instance with a second `singlethread` instance, next to the one we initialize in the `main.go` file...